src/main/resources/assets/admin/common/styles/api/form/set/form-set-view.less

```diff
@@ -1,205 +1,215 @@
 @occurrence-label-margin-left: 10px;
 
-.form-item-set-view {
-  > .occurrence-views-container > .form-item-set-occurrence-view {
-    padding: 10px;
-  }
+.form-view {
+  .form-item-set-view {
+    > .occurrence-views-container > .form-item-set-occurrence-view {
+      padding: 13px;
+    }
 
-  &.max-1-occurrence {
     .form-item-set-occurrence-view {
-
-      > .form-item-set-occurrences-container {
-        padding: 0 10px;
+      .form-occurrence-draggable-label {
+        padding-right: 20px;
+        width: calc(100% - 20px);
+        .ellipsis();
       }
     }
   }
 
-  .form-item-set-occurrence-view {
-    .form-occurrence-draggable-label {
-      padding-right: 20px;
-      cursor: zoom-out;
-      .ellipsis();
-    }
-
-    &.collapsed .form-occurrence-draggable-label {
-      cursor: zoom-in;
-    }
-  }
-}
-
-.form-option-set-view .form-item-set-view {
-  > .occurrence-views-container > .form-item-set-occurrence-view {
-    padding-bottom: 20px;
-  }
-}
-
-.form-item-set-view, .form-option-set-view {
+  .form-item-set-view, .form-option-set-view {
+    .occurrence-views-container {
 
-  &:not(.max-1-occurrence) {
-    > .occurrence-views-container {
-      background-color: @admin-bg-light-gray;
-      padding: 10px 10px 0 10px;
-      .clearfix();
+      &:empty {
+        display: none;
+      }
 
-      &:not(:empty) + .bottom-button-row {
-        padding-top: 0;
-        padding-bottom: 5px;
+      .form-option-set-occurrence-view {
+        padding: 10px;
       }
-    }
-  }
 
-  .form-item-set-drop-target-placeholder, .form-option-set-drop-target-placeholder {
-    border: 2px dashed #000;
-    padding: 14px;
-    color: #000;
-    font-size: 14px;
-    text-align: center;
-    margin: 10px;
-  }
+      .form-item-set-occurrence-view, .form-option-set-occurrence-view {
+        clear: left;
+        margin: 0;
+        box-shadow: 0 1px 2px 1px rgba(0, 0, 0, 0.2);
 
-  .input-view {
-    .input-label {
-      max-width: 90px;
-      margin-right: 10px;
-    }
+        &.collapsed {
+          > .form-occurrence-draggable-label {
+            cursor: zoom-in;
+          }
+        }
 
-    .input-type-view,
-    .bottom-button-row,
-    .validation-viewer {
-      margin-left: @form-label-width;
-    }
+        & > .remove-button {
+          &:extend(.icon);
+          .icon-close();
+          .focusable();
+
+          margin-top: 3px;
+          margin-right: -4px;
+          border: 1px solid transparent;
+          line-height: 12px;
+          float: right;
+          text-decoration: none;
+          cursor: pointer;
+        }
 
-    &.label-over-input {
-      .input-label {
-        max-width: 100%;
-      }
-      .input-type-view,
-      .bottom-button-row,
-      .validation-viewer {
-        margin-left: 0;
-      }
-    }
-  }
+        .form-occurrence-draggable-label {
```

Comment:
Codacy found an issue: [Expected selector ".form-item-set-view .occurrence-views-container .form-item-set-occurrence-view .form-occurrence-draggable-label" to come before selector ".form-item-set-view .occurrence-views-container .form-item-set-occurrence-view.collapsed .form-occurrence-draggable-label" (no-descending-specificity)](https://app.codacy.com/manual/enonic/lib-admin-ui/pullRequest?prid=6422415)